### DISCUSSION TITLE
https://dbeaver.io/contribution/ - hard to find on main site

### DISCUSSION THREAD

**Discussion by nmat** [dbeaver/dbeaver#11604] [Ideas]
#### Describe the problem you're observing:
In order to find this link you have to go and search for Contribute or Donate otherwise it is hard to find this link on the page.

I have tried to find any reference on page to this specific Contribution page but can't find it.

Might be good to add this somewhere to get more people to note on donate and more on that page.

**Comment by agarcialeon** [comment]
I haven't found any link to that page too. There are more links referencing Github Issues than the page you mentioned. It seems that the team forgot to link it in the main menu of they decided to link directly to the Github repo without mentioning the donation button.

**Comment by LonwoLonwo** [comment]
Thanks, guys! 
But we like everything as it is. 
If you really want to help us, then you can buy [EE license](https://dbeaver.com/buy/).
